projects/fusion-ui/components/chip-filters/common/base/chip-filters.base.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="chip-filters-container">
     <ng-content></ng-content>
-    <ng-container *ngIf="showAddFilter$ | async">
+    <ng-container *ngIf="(options$ | async).length">
```

Comment:
add optional chaining like  this
 (options$ | async)?.length

Reply:
Ok.